### ISSUE TITLE
Long Run Time for Bunny

### ISSUE THREAD

**Issue by EdwinKLee** [alecjacobson/computer-graphics-ray-casting#12]
After comparing with a few classmates, it seems we're all taking at least 5-10 minutes to generate the bunny image (even on decently high end desktops). In README.html it says this should only take a few seconds, so we're wondering if this is correct. We're also concerned because the ray tracing lab is taking over an hour to run. 

Is this just a typo, or are we missing something?

**Comment by alecjacobson** [comment]
In Debug or Release mode?

**Comment by chenbowen** [comment]
On my desktop, it takes less than 30s in Release mode.

**Comment by EdwinKLee** [comment]
Yikes I neglected to specify the mode for CMake, thank you!

**Comment by shanchao95** [comment]
Hi, when i used the release mode and make, got an error message, Thanks in advance.

[ 22%] Linking CXX executable raycasting
Undefined symbols for architecture x86_64:
  "Eigen::MatrixBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::cross_product_return_type<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::type Eigen::MatrixBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> >::cross<Eigen::Matrix<double, 3, 1, 0, 3, 1> >(Eigen::MatrixBase<Eigen::Matrix<double, 3, 1, 0, 3, 1> > const&) const", referenced from:
      Triangle::intersect(Ray const&, double, double&, Eigen::Matrix<double, 3, 1, 0, 3, 1>&) const in Triangle.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [raycasting] Error 1
make[1]: *** [CMakeFiles/raycasting.dir/all] Error 2
make: *** [all] Error 2"

**Comment by alecjacobson** [comment]
You're probably missing a `#include <Eigen/Geometry>` in your Triangle.cpp 

(this is a weird Eigen problem).

**Comment by shanchao95** [comment]
right, I have #include <Eigen/Core>,but missed #include <Eigen/Geometry>, thanks!!

**Comment by velociraptor111** [comment]
Same problem with @shanchao95 .. I just realized I have this problem when I tried running 'make' on the release build version. But it was always **fine**  when I had it running on debug mode. 

Since the deadline to submit has passed, I did not do the one line fix the in the code triangle.cpp as mentioned by @alecjacobson . 

When the TA is running the code on their local machine will they be having this problem as well? Will they be running on debug or release version ? Will I lose mark for this? @psarahdactyl
